zerver/lib/error_notify.py

```diff
@@ -23,8 +23,8 @@ def logger_repr(report: Dict[str, Any]) -> str:
     return "Logger {logger_name}, from module {log_module} line {log_lineno}:".format(**report)
 
 def user_info_str(report: Dict[str, Any]) -> str:
-    if report['user_full_name'] and report['user_email']:
-        user_info = "{user_full_name} ({user_email})".format(**report)
+    if report['user'].get('user_full_name'):
+        user_info = "{user[user_full_name]} ({user[user_email]}) ({user[user_role]})".format(**report)
```

Comment:
While we're in here, can we make this consistent with below -- using `<brackets>` for the email?